examples/wcf/server-netframework/Examples.Wcf.Server.NetFramework.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <TargetFramework>net462</TargetFramework>
```

Comment:
I hope it will be working:

```suggestion
    <TargetFramework>$(NetFrameworkMinimumSupportedVersion)</TargetFramework>
```

alternatively, you can create 

`<DefaultTargetFrameworkForExampleAppsForNetFramework>$(NetFrameworkMinimumSupportedVersion)<DefaultTargetFrameworkForExampleAppsForNetFramework>` and use it here.

Reply:
Thank you for the suggestion.